packages/create-cloudflare/turbo.json

```diff
@@ -1,13 +1,28 @@
 {
 	"$schema": "http://turbo.build/schema.json",
 	"extends": ["//"],
-	"pipeline": {
+	"tasks": {
 		"build": {
-			"env": ["TEST_PM", "TEST_PM_VERSION", "npm_config_user_agent", "CI"],
+			"env": [
+				"TMPDIR",
```

Comment:
turbo needs you to whitelist any nv vars that you'd use. We have to add these new ones so that os.tmpdir() works correctly (which depend on them)